scripts/sanitize.sh

```diff
@@ -8,12 +8,39 @@ set -o pipefail
 Rebuilds the code with the sanitizers and runs the tests
 
 '
- # Set up the environment by installing mason and clang++
- # See https://github.com/mapbox/node-cpp-skel/blob/master/docs/extended-tour.md#configuration-files
-./scripts/setup.sh --config local.env
-source local.env
+
+# See https://github.com/mapbox/node-cpp-skel/blob/master/docs/extended-tour.md#configuration-files
+
 make clean
-export CXXFLAGS="${MASON_SANITIZE_CXXFLAGS} ${CXXFLAGS:-}"
+
+# https://github.com/google/sanitizers/wiki/AddressSanitizerAsDso
+SHARED_LIB_EXT=.so
+if [[ $(uname -s) == 'Darwin' ]]; then
+    SHARED_LIB_EXT=.dylib
+fi
+
+if [[ $(uname -s) == 'Linux' ]]; then
+    ./node_modules/.bin/mason-js install binutils=2.30 --type=compiled
+    ./node_modules/.bin/mason-js link binutils=2.30 --type=compiled
+    export PATH=$(pwd)/mason_packages/.link/bin:${PATH}
+fi
+export MASON_LLVM_RT_PRELOAD=$(pwd)/$(ls mason_packages/.link/lib/clang/*/lib/*/libclang_rt.asan*${SHARED_LIB_EXT})
```

Comment:
Awesome 🙌  nice to keep these settings tightly scoped to `sanitize` instead of in the general `setup.sh` file.

Reply:
Right? This is such a nice cleanup that mason-js inspired/allowed/unlocked.